src/CommunityToolkit.Aspire.OllamaSharp/AspireOllamaSharpExtensions.cs

```diff
@@ -10,7 +11,7 @@ namespace Microsoft.Extensions.Hosting;
 /// </summary>
 public static class AspireOllamaSharpExtensions
 {
-    private const string DefaultConfigSectionName = "Aspire:OllamaSharp";
+    private const string DefaultConfigSectionName = "Aspire:Ollama";
```

Comment:
This needs to stick with `OllamaSharp` as that's the client this is relevant for